Dockerfile.jsonschema

```diff
@@ -0,0 +1,8 @@
+# 3.18.2
+FROM alpine@sha256:25fad2a32ad1f6f510e528448ae1ec69a28ef81916a004d3629874104f8a7f70
+RUN apk add --update protoc=3.21.12-r2 protobuf-dev=3.21.12-r2 go=1.20.5-r0 git=2.40.1-r0
+RUN go install github.com/chrusty/protoc-gen-jsonschema/cmd/protoc-gen-jsonschema@1.4.1
+# This is required to get the field_behavior.proto file
+# NOTE: --filter=tree:0 performs a treeless clone; we do this to optimize cloning
+# this otherwise relatively heavy repository.
+RUN git clone --filter=tree:0 https://github.com/googleapis/googleapis.git
```

Comment:
NB: This repository doesn't appear to be versioned in any way, and isn't packaged in alpine. I could check out a fixed revision if we're like more consistency here.

Reply:
I'd like to pin it to a fixed revision.

Reply:
Hmm, this is moderately annoying: we can't do a fast-path `clone` with a specific revision. I'll see about just downloading it directly as a ZIP or similar and unarchiving.